src/vm/dwreport.cpp

```diff
@@ -2426,9 +2440,11 @@ FaultReportResult DoFaultReportWorker(      // Was Watson attempted, successful?
     pWatsonSharedMemory->bfmsoctdsLetRun = offerFlags;
 
     {
+        PathString wzModuleFileName;
         DWORD dwRet = WszGetModuleFileName(NULL,
-	                                       pWatsonSharedMemory->wzModuleFileName,
-	                                       NumItems(pWatsonSharedMemory->wzModuleFileName));
+	                                       wzModuleFileName);
+        wcsncpy_s(pWatsonSharedMemory->wzModuleFileName, NumItems(pWatsonSharedMemory->wzModuleFileName), wzModuleFileName, _TRUNCATE);
```

Comment:
I'm not certain offhand, but I'm sure we could find out with some research. Before spending time doing that however, is there a scenario that will give us a long file path here? The RunWatson function is downlevel WER integration for Vista and below. My understanding (which might be entirely wrong) is that the long file name support wasn't added until windows 8 timeframe.